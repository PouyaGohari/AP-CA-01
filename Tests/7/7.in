6
pa1 pa2 pa3 pa4
pa1:0 YES|1 NO
pa2:1 IF_NECESSARY|5 NO
pa3:4 YES|2 IF_NECESSARY
pa4:2 IF_NECESSARY|3 NO