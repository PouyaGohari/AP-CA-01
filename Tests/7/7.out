Time slot 0:
YES: pa1 
NO: 
IF_NECESSARY: pa2 pa3 pa4 
###
Time slot 4:
YES: pa3 
NO: 
IF_NECESSARY: pa1 pa2 pa4 
###
Time slot 2:
YES: 
NO: 
IF_NECESSARY: pa1 pa2 pa3 pa4 
###
